cmd/iop/install.go

```diff
@@ -50,7 +50,7 @@ func installManifests(args *rootArgs, logOpts *log.Options) {
 		logAndFatalf(args, "Could not generate manifest: %v", err)
 	}
 
-	out, err := manifest.ApplyAll(manifests, version.NewVersion("", 1, 2, 0, ""), args.dryRun, args.verbose)
+	out, err := manifest.ApplyAll(manifests, version.NewVersion(1, 2, 0, ""), args.dryRun, args.verbose)
```

Comment:
Should this be 1.3.0 as well?

Reply:
Actually it should be using the top level version definition. I've removed this hardcoding, here and elsewhere.